### PR TITLE
Android US election notification format

### PR DESCRIPTION
**Pull request by davidfurey**
@maxspencer

### CODE REVIEW

File: notification/app/notification/models/android/Keys.scala [hunk 1]
```diff
@@ -28,4 +29,6 @@ object Keys {
   final val AwayTeamScore = "AWAY_TEAM_SCORE"
   final val Topics = "topics"
   final val UniqueIdentifier = "uniqueIdentifier"
+  final val electoralVotesAvailable = "electoralVotesAvailable"
```
Comment (maxspencer) [inline notification/app/notification/models/android/Keys.scala]: @davidfurey you had a good suggestion to change this to `"electoralCollegeSize"` can we still do that? Also nitpick: should be `Electoral...` to match the names of the other keys.

**Comment by alexduf** [comment]
Still looking good to me

File: notification/test/notification/models/azure/iOSNotificationSpec.scala [hunk 2]
```diff
@@ -234,37 +234,65 @@ class iOSNotificationSpec extends Specification with Mockito {
...
+        sound = None
       ),
-      customProperties = Map(
-        "t" -> "e"
+      customProperties = StandardProperties(
+        t = "us-election",
+        election = Some(ElectionProperties(
+          title = "Live election results",
+          body = "\u2022 Electoral votes: Clinton 220, Trump 133\n\u2022 270 electoral votes to win\n• 35 states called, 5 swing states (OH, PA, NV, CO, FL)\n• Popular vote: Clinton 52%, Trump 43% with 42% precincts reporting",
+          richviewbody = "• 35 states called, 5 swing states (OH, PA, NV, CO, FL)\n• Popular vote: Clinton 52%, Trump 43% with 42% precincts reporting",
```
Comment (fgauchet) [inline notification/test/notification/models/azure/iOSNotificationSpec.scala]: Should you have the encoded unicode character here, and in the line above?

File: notification/test/notification/models/azure/iOSNotificationSpec.scala [hunk 3]
```diff
@@ -234,37 +234,65 @@ class iOSNotificationSpec extends Specification with Mockito {
...
+      )),
       topic = Set.empty
     )
 
     val push = Push(notification, Left(Set(Topic(TagSeries, "series-a"), Topic(TagSeries, "series-b"))))
 
     val expected = Body(
       aps = APS(
-        alert = Some(Right("test")),
+        alert = None,
```
Comment (fgauchet) [inline notification/test/notification/models/azure/iOSNotificationSpec.scala]: Does this mean the key will not be present at all in the payload?

Comment (davidfurey) [inline notification/test/notification/models/azure/iOSNotificationSpec.scala]: Yes.  If the value is None then that key/value will not be included.

File: notification/app/notification/models/android/Notification.scala [hunk 4]
```diff
@@ -119,13 +120,38 @@ case class GoalAlertNotification(
...
   def payload: Map[String, String] = Map(
     Keys.Type -> `type`,
     Keys.UniqueIdentifier -> id.toString,
-    Keys.Message -> message,
-    Keys.Debug -> debug.toString
-  )
+    Keys.ExpandedMessage -> expandedMessage,
+    Keys.ShortMessage -> shortMessage,
+    Keys.Debug -> debug.toString,
+    Keys.ElectoralCollegeSize -> "538",
```
Comment (alexduf) [inline notification/app/notification/models/android/Notification.scala]: Do we need to send that constant?

Comment (davidfurey) [inline notification/app/notification/models/android/Notification.scala]: Yes: https://github.com/guardian/android-news-app/blob/8823a761b7d89e554d95fb51ec1d8b3649513828/android-news-app/src/main/java/com/guardian/gcm/data/LiveElectionsData.java#L87

**Pull request merged**